### PR TITLE
Fixes button padding

### PR DESCRIPTION
**Pull request by cquiroz**
The tighter padding for forms was affecting elements beyond just forms.
This PR fixes it and adds a few other minor touches

### CODE REVIEW

**Comment by toddburnside** [comment]
So, inside a form, the buttons will have reduced margins, but will have the original margins outside of a form?

**Comment by cquiroz** [comment]
Yes, we may need to do the same for other buttons but I didn't want a blanket change affecting all buttons

**Comment by toddburnside** [comment]
LGTM. I'll back out the change I had that required setting a class on buttons you wanted "fully padded".

**Review by toddburnside — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -15,7 +15,7 @@ ThisBuild / useLog4J := true
 
 addCommandAlias(
   "exploreWDS",
-  "; explore/fastOptJS::stopWebpackDevServer; explore/fastOptJS::startWebpackDevServer; ~explore/fastOptJS"
+  "; stopWDS; explore/fastOptJS::startWebpackDevServer; ~explore/fastOptJS"
```
Comment (rpiaggio) [inline build.sbt:18]: I think this won't work as expected if you are standing in another subproject.

File: build.sbt [hunk 2]
```diff
@@ -210,21 +210,26 @@ def depsResourceDirsTask(conf: ConfigKey): sbt.Def.Initialize[sbt.Task[Seq[java.
   }
 
 val clueGenEq =
-  settingKey[Boolean]("Include cats.Eq instances for classes generated by clue macros.")
+  settingKey[Boolean]("Include cats.Eq instances for classes generated by clue macros.").withRank(
+    KeyRanks.Invisible
```
Comment (rpiaggio) [inline build.sbt:214]: Thank you and sorry for the warnings :)

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**